### PR TITLE
Installed Good_Job for async jobs

### PR DESCRIPTION
**Pull request by wJoenn**
It's an alternative to Sidekiq
Sidekiq requires Redis which means 5$ a month on heroku, this one uses Postgresql so free because we already use it.
It's slower but who cares we're only gonna send 5 mails a day maximum considering Chloe's market

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
   root to: "pages#home"
 
+  authenticate :user, ->(user) { user.admin? } do
+    mount GoodJob::Engine => "good_job"
+  end
```
Comment (cipriancrisan) [inline config/routes.rb:6]: My understanding is that GoodJob is used to manage the background jobs, like emailing.
So in this block of code I understand that the program mounts the GoobJob engine only if the user is the admin.
Do we want to launch the background jobs in a queue only when the admin says 'go' in some way ?

File: app/models/user.rb [hunk 2]
```diff
@@ -37,7 +28,16 @@ def self.from_omniauth(auth)
     User.create!(email:, password:, first_name:, last_name: last_name.join(" "), provider: auth.provider, uid:)
   end
```
Comment (cipriancrisan) [inline app/models/user.rb:29]: we use the function `self.from_omniauth(auth)` to handle the Facebook authentication right?
so we create variables to save the user's attributes from the auth object ?

File: config/routes.rb (hunk 1 above)
Comment (wJoenn) [reply config/routes.rb:6]: This the routes folder
It means this is a path to some url endpoint.
In this case it's wesbite/good_job

What this all means is that only the admin can access this page

Comment (wJoenn) [reply config/routes.rb:6]: It's a dashboard provided by the gem to monitor background tasks
image

**Pull request merged**